ark/schema/shared/intersections.ts

```diff
@@ -201,7 +201,7 @@ const _pipeMorphed = (
 
 		return ctx.$.node("morph", {
 			morphs,
-			in: from.in
+			in: from.inner.in as any
```

Comment:
I would usually use `as never` in a case like this but I can update that later, it's pretty inconsequential.

Reply:
Please check the types, there's a good chance it shouldn't need the case at all